doap.rdf

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
I don't think we need these `---` I think they are the "front matter" from the jekyll system that arrow-site uses